cylc/flow/etc/tutorial/cylc-forecasting-workflow/bin/get-observations

```diff
@@ -73,19 +73,6 @@ class NoDataException(Exception):
     ...
 
 
-def get_datapoint_data(site_id, time, api_key):
-    """Get weather data from the DataPoint service."""
-    # The URL required to get the data.
-    time = datetime.strptime(time, '%Y%m%dT%H%MZ').strftime('%Y-%m-%dT%H:%MZ')
-    url = BASE_URL.format(time=time, site_id=site_id, api_key=api_key)
-    req = requests.get(url)
-    if req.status_code != 200:
-        raise Exception(f'{url} returned exit code {req.status_code}')
-    # Get the data and parse it as JSON.
-    print('Opening URL: %s' % url)
-    return req.json()['SiteRep']['DV']['Location']
-
-
 def get_archived_data(site_id, time):
```

Comment:
I can't see any evidence of actual archived data - Perhaps we should save some at some point?